app/src/main/java/io/intrepid/skeleton/base/BaseFragment.java

```diff
@@ -0,0 +1,92 @@
+package io.intrepid.skeleton.base;
+
```

Comment:
This can be broken up into a BaseFragment that handles instantiating a view with butterknife and a BaseFragmentMVP that handles creating a presenter. You may not want to do this since this class will force all children to inherit an MVP style. You may want to do this because occasionally a fragment just wants to display a view without much of a data request and is already very testable without separating data from the view.
check out -->
(https://github.com/IntrepidPursuits/MITMobius-android/blob/develop/app/src/main/java/edu/mit/mobius/app/screens/BaseFragment.java) 
(https://github.com/IntrepidPursuits/MITMobius-android/blob/develop/app/src/main/java/edu/mit/mobius/app/screens/BaseMVPFragment.java)